mvrx-navigation/build.gradle

```diff
@@ -0,0 +1,56 @@
+import com.airbnb.mvrx.JacocoReportTask
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'jacoco'
+apply plugin: 'com.vanniktech.maven.publish'
+
+android {
+    resourcePrefix "mvrx_"
+
+    sourceSets {
+        test.res.srcDirs += 'src/test/res'
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            all {
+                jacoco {
+                    includeNoLocationClasses = true
+                }
+            }
+        }
+    }
+}
+
+dependencies {
+
+    implementation project(":mvrx")
+    implementation project(":testing")
+    implementation Libraries.lifecycleCommon
+
+    testImplementation Libraries.rxAndroid
+    testImplementation TestLibraries.junit
+    testImplementation TestLibraries.mockito
+    testImplementation TestLibraries.roboeletric
+    testAnnotationProcessor "com.google.auto.service:auto-service:1.0-rc4"
```

Comment:
Could you update these and the other dependencies to use the Libraries and TestLibraries objects?